SPECS-EXTENDED/gdal/gdal.spec

```diff
@@ -0,0 +1,1255 @@
+Summary:        GDAL file format library
+# See frmts/grib/degrib/README.TXT
+Provides:       bundled(g2lib) = 1.6.0
+Provides:       bundled(degrib) = 2.14
+
+%description libs
+This package contains the GDAL file format library.
+
+# No complete java yet in EL8
+%if 0%{?with_java}
+%package java
+Summary:        Java modules for the GDAL file format library
+Requires:       %{name}-libs%{?_isa} = %{version}-%{release}
+Requires:       jpackage-utils
+
+%description java
+The GDAL Java modules provide support to handle multiple GIS file formats.
+
+%package javadoc
+Summary:        Javadocs for %{name}
+Requires:       jpackage-utils
+BuildArch:      noarch
+
+%description javadoc
+This package contains the API documentation for %{name}.
+%endif
+
+
+%if 0%{?with_python3}
+%package -n python3-gdal
+Summary:        Python modules for the GDAL file format library
+%{?python_provide:%python_provide python3-gdal}
+Requires:       %{name}-libs%{?_isa} = %{version}-%{release}
+Requires:       python3-numpy
+
+%description -n python3-gdal
+The GDAL Python 3 modules provide support to handle multiple GIS file formats.
+
+%package python-tools
+Summary:        Python tools for the GDAL file format library
+Requires:       python3-gdal
+
+%description python-tools
+The GDAL Python package provides number of tools for programming and
+manipulating GDAL file format library
+
+# We don't want to provide private Python extension libs
+%global __provides_exclude_from ^%{python3_sitearch}/.*\.so$
+%endif
+
+
+%prep
+%autosetup -N -p1 -n %{name}-%{version}
+
+# Delete bundled libraries
+rm -rf frmts/zlib
+rm -rf frmts/png/libpng
+rm -rf frmts/gif/giflib
+rm -rf frmts/jpeg/libjpeg
+rm -rf frmts/jpeg/libjpeg12
+rm -rf frmts/gtiff/libgeotiff
+rm -rf frmts/gtiff/libtiff
+rm -rf mrf/LERCV1
+rm -rf third_party/LercLib
+
+# Setup autotest directory
+tar xf %{SOURCE1}
+mv %{name}autotest-%{version} autotest
+
+# Need to patch autotest
+%autopatch -p1
+
+
+%build
+%cmake \
+  -DCMAKE_INSTALL_INCLUDEDIR=include/gdal \
+  -DGDAL_JAVA_INSTALL_DIR=%{_jnidir}/%{name} \
+  -DGDAL_USE_JPEG12_INTERNAL=OFF \
+%cmake_build
+
+%install
+%cmake_install
+
+
+# List of manpages for python scripts
+for file in %{buildroot}%{_bindir}/*.py; do
+  if [ -f %{buildroot}%{_mandir}/man1/`basename ${file/.py/.1*}` ]; then
+    echo "%{_mandir}/man1/`basename ${file/.py/.1*}`" >> gdal_python_manpages.txt
+    echo "%exclude %{_mandir}/man1/`basename ${file/.py/.1*}`" >> gdal_python_manpages_excludes.txt
+  fi
+done
+
+# Multilib
+# - cpl_config.h is arch-dependent (contains various SIZEOF defines)
+# - gdal-config stores arch-specific information
+mv %{buildroot}%{_includedir}/%{name}/cpl_config.h %{buildroot}%{_includedir}/%{name}/cpl_config-%{cpuarch}.h
+cp -a %{SOURCE2} %{buildroot}%{_includedir}/%{name}/cpl_config.h
+mv %{buildroot}%{_bindir}/%{name}-config %{buildroot}%{_bindir}/%{name}-config-%{cpuarch}
+cp -a %{SOURCE3} %{buildroot}%{_bindir}/%{name}-config
+
+
+%if 0%{run_tests}
```

Comment:
I think we should drop this conditional.
We have `RUN_CHECK=y/n` to run this conditionally.

Also, during tests internet is availible, we might want to install `pytest>=6.0.0` and ensure we do not skip any potential tests.

Reply:
I ran with pytest, some of the auto (python) based tests themself are not correct, which is why I have ignored them as mentioned in the changelog(updated).